packages/google-cloud-quotas/README.rst

```diff
@@ -10,10 +10,10 @@ Python Client for Cloud Quotas API
 
 .. |preview| image:: https://img.shields.io/badge/support-preview-orange.svg
    :target: https://github.com/googleapis/google-cloud-python/blob/main/README.rst#stability-levels
-.. |pypi| image:: https://img.shields.io/pypi/v/google-cloud-cloudquotas.svg
-   :target: https://pypi.org/project/google-cloud-cloudquotas/
-.. |versions| image:: https://img.shields.io/pypi/pyversions/google-cloud-cloudquotas.svg
-   :target: https://pypi.org/project/google-cloud-cloudquotas/
+.. |pypi| image:: https://img.shields.io/pypi/v/google-cloud-quotas.svg
+   :target: https://pypi.org/project/google-cloud-quotas/
+.. |versions| image:: https://img.shields.io/pypi/pyversions/google-cloud-quotas.svg
+   :target: https://pypi.org/project/google-cloud-quotas/
 .. _Cloud Quotas API: https://cloud.google.com/docs/quota/api-overview
 .. _Client Library Documentation: https://cloud.google.com/python/docs/reference/google-cloud-cloudquotas/latest/summary_overview
```

Comment:
do we want to update the link for the client library documentation?

Reply:
Yes, I filed https://github.com/googleapis/google-cloud-python/issues/13186 to follow up on this issue as a similar problem exists in other packages where the PyPI package name is not part of the url